src/sparql/rm_xref_by_prefix.ru

```diff
@@ -16,5 +16,5 @@ WHERE {
     oboInOwl:hasDbXref ?xref ;
     ?p1 ?o2 .
   }
-  FILTER( STRSTARTS(str(?xref), "UMLS_ICD9CM_2005_AUI:") || STRSTARTS(str(?xref), "ICD11:") || STRSTARTS(str(?xref), "SNOMEDCT_US_") || STRSTARTS(str(?xref), "IMDRF:") || STRSTARTS(str(?xref), "url:") )
+  FILTER( STRSTARTS(str(?xref), "UMLS_ICD9CM_2005_AUI:") || STRSTARTS(str(?xref), "SNOMEDCT_US_") || STRSTARTS(str(?xref), "IMDRF:") || STRSTARTS(str(?xref), "url:") )
```

Comment:
## Merging into `develop` or `main`
@hrshdhgd I see that this is merging into main, but basically from now on, per #405 (which isn't completed yet) we're going to follow the normal git flow of `main` <-- `develop` <-- feature branches.

We could change the base on this PR but IDK how much it matters. I update `develop` from master regularly as needed anyway.

Reply:
@hrshdhgd I approve as well as Nico. Prolly good to merge. I can update `develop` later.